### PR TITLE
Update to Swift 5.0

### PR DESCRIPTION
**Pull request by vkcldhkd**
Update to Swift 5.0 :)

### CODE REVIEW

File: CenteredCollectionView/Classes/CenteredCollectionViewFlowLayout.swift [hunk 1]
```diff
@@ -31,7 +31,10 @@ open class CenteredCollectionViewFlowLayout: UICollectionViewFlowLayout {
 			return itemSize.width + minimumLineSpacing
 		case .vertical:
 			return itemSize.height + minimumLineSpacing
-		}
+
```
Comment (BenEmdon) [inline CenteredCollectionView/Classes/CenteredCollectionViewFlowLayout.swift]: Nitpick: remove this extra newline
```suggestion
```

File: CenteredCollectionView/Classes/CenteredCollectionViewFlowLayout.swift [hunk 2]
```diff
@@ -31,7 +31,10 @@ open class CenteredCollectionViewFlowLayout: UICollectionViewFlowLayout {
 			return itemSize.width + minimumLineSpacing
 		case .vertical:
 			return itemSize.height + minimumLineSpacing
-		}
+            
+        default:
+            return 0
+        }
```
Comment (BenEmdon) [inline CenteredCollectionView/Classes/CenteredCollectionViewFlowLayout.swift]: The spacing here doesn't match the rest of the code. Would you mind making the stacking consistent with the rest of the file?

File: CenteredCollectionView/Classes/CenteredCollectionViewFlowLayout.swift [hunk 3]
```diff
@@ -70,6 +73,10 @@ open class CenteredCollectionViewFlowLayout: UICollectionViewFlowLayout {
 				let inset = (currentCollectionViewSize.height - itemSize.height) / 2
 				collectionView.contentInset = UIEdgeInsets(top: inset, left: 0, bottom: inset, right: 0)
 				collectionView.contentOffset = CGPoint(x: 0, y: -inset)
+                
+            default:
+                collectionView.contentInset = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 0)
+                collectionView.contentOffset = .zero
```
Comment (BenEmdon) [inline CenteredCollectionView/Classes/CenteredCollectionViewFlowLayout.swift]: The spacing here doesn't match the rest of the code. Would you mind making the stacking consistent with the rest of the file?

File: CenteredCollectionView/Classes/CenteredCollectionViewFlowLayout.swift [hunk 4]
```diff
@@ -111,6 +118,9 @@ open class CenteredCollectionViewFlowLayout: UICollectionViewFlowLayout {
 				newOffset += velocity.y > 0 ? pageHeight : -pageHeight
 			}
 			return CGPoint(x: proposedContentOffset.x, y: newOffset)
+            
+        default:
+            return .zero
```
Comment (BenEmdon) [inline CenteredCollectionView/Classes/CenteredCollectionViewFlowLayout.swift]: The spacing here doesn't match the rest of the code. Would you mind making the stacking consistent with the rest of the file?

File: CenteredCollectionView/Classes/CenteredCollectionViewFlowLayout.swift [hunk 5]
```diff
@@ -133,6 +143,10 @@ open class CenteredCollectionViewFlowLayout: UICollectionViewFlowLayout {
 			let pageOffset = CGFloat(index) * pageWidth - collectionView.contentInset.top
 			proposedContentOffset = CGPoint(x: collectionView.contentOffset.x, y: pageOffset)
 			shouldAnimate = abs(collectionView.contentOffset.y - pageOffset) > 1 ? animated : false
+            
+        default:
+            proposedContentOffset = .zero
+            shouldAnimate = false
```
Comment (BenEmdon) [inline CenteredCollectionView/Classes/CenteredCollectionViewFlowLayout.swift]: The spacing here doesn't match the rest of the code. Would you mind making the stacking consistent with the rest of the file?

File: CenteredCollectionView/Classes/CenteredCollectionViewFlowLayout.swift [hunk 6]
```diff
@@ -148,6 +162,8 @@ private extension CenteredCollectionViewFlowLayout {
 			origin = CGPoint(x: proposedContentOffset.x, y: collectionView.contentOffset.y)
 		case .vertical:
 			origin = CGPoint(x: collectionView.contentOffset.x, y: proposedContentOffset.y)
+        default:
+            origin = .zero
```
Comment (BenEmdon) [inline CenteredCollectionView/Classes/CenteredCollectionViewFlowLayout.swift]: The spacing here doesn't match the rest of the code. Would you mind making the stacking consistent with the rest of the file?

File: CenteredCollectionView/Classes/CenteredCollectionViewFlowLayout.swift [hunk 7]
```diff
@@ -166,6 +182,8 @@ private extension CenteredCollectionViewFlowLayout {
 			proposedCenterOffset = proposedContentOffset.x + collectionView.bounds.size.width / 2
 		case .vertical:
 			proposedCenterOffset = proposedContentOffset.y + collectionView.bounds.size.height / 2
+        default:
+            proposedCenterOffset = .zero
```
Comment (BenEmdon) [inline CenteredCollectionView/Classes/CenteredCollectionViewFlowLayout.swift]: The spacing here doesn't match the rest of the code. Would you mind making the stacking consistent with the rest of the file?

**Comment by vkcldhkd** [comment]
Below are excerpts from https://docs.swift.org/swift-book/LanguageGuide/ControlFlow.html :)

"Every switch statement must be exhaustive. That is, every possible value of the type being considered must be matched by one of the switch cases. If it’s not appropriate to provide a case for every possible value, you can define a default case to cover any values that are not addressed explicitly. This default case is indicated by the default keyword, and must always appear last."

**Pull request merged**